R/pkg/R/DataFrame.R

```diff
@@ -501,6 +501,72 @@ setMethod("createOrReplaceTempView",
               invisible(callJMethod(x@sdf, "createOrReplaceTempView", viewName))
           })
 
+#' Creates a global temporary view using the given name.
+#'
+#' Creates a new global temporary view using a SparkDataFrame in the Spark Session.
+#' Global temporary view is cross-session while a temporary view is not. Its lifetime
+#' is the lifetime of the Spark application, i.e. it will be automatically dropped
+#' when the application terminates. It's tied to a system preserved database `global_temp`,
+#' and we must use the qualified name to refer a global temporary view, e.g.
+#' `SELECT * FROM global_temp.view1`. If a global temporary view with the same name
+#' already exists, it raises exception.
```

Comment:
I guess `` ` ... ` `` should be `\code{ ... }` if this is meant to be a code block.

BTW, I hope the documentations for the same API in Scala/Python/R are almost identical if these are not language-specific. This requires a bit of overhead when someone sweep them to fix the same contents and sometimes it needs several follow-up PRs.